.azurepipelines/Platform-Build-GCC5.yml

```diff
@@ -88,6 +116,38 @@ jobs:
           **/*/*TestApp.efi
         BuildArtifactsOther: "**/unit_test_results/*"
 
+      QemuQ35x86_DEBUG_ARM:

```

Comment:
To reduce pipelines, maybe we don't care about the IA32,X64 build on ARM?

Reply:
Yep, removed.